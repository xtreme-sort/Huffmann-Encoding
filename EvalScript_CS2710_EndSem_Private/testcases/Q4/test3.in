GenerateLengthLimitedHuffman
kknoljdfrjluzrcsvlolfuwoewsikgbvqtuanuownnhgqchajpfbavvechaftpmvhtozvbqnyhcwawoxfvxevhmchfkzjtbpasjcpwbaggutptuoewpvnztavnopbbxybnirdbfxaapwsynlnleijeenoqdoialhrazxwqbqtetqznhotzzbsjrszitoxmkswzlyycohtutmxtedimkahsctviiwfmuzbmrlimdilzswraiogrdcurgkymkkaxtrbarsrilxvuicpekdtvznlrigeugztyahqazmpvvfgtzypbvgtugtaqbfzawyojekjgsmkshmtupwcmprkkqajwqlmbdvgktnmwecyslctfknxpdciodclfnlngnqwiqrselanqbzzrmboiyvnwwzkiyyuqrebovvmysyyuewdeiadfzlptqhhedxesawisinureoyvavcsdrkytpdwroykpqqqdsrgyabgrnsvdvcerxehyonfrukhhlzvygopzkuldokdlvbxlfbbhglqpxhegqpfngsdelmgcpcfperaehnflopwngezlopbrwywcseejmdtcxxbnewaytjviavorxszmvztllymcskzldiwjupufvusbtniioejpmldegfardkjaclmmekqaojerlcmvjgubstpfyfvcikholsnxrwoqwhenbphtyagrd
9
Quit
