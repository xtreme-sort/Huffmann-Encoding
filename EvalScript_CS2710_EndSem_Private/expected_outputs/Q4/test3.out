a 0000
e 0001
l 0010
r 0011
t 0100
v 0101
b 01100
c 01101
d 01110
f 01111
g 10000
h 10001
i 10010
j 10011
k 10100
m 10101
n 10110
o 10111
p 11000
q 11001
s 11010
u 11011
w 11100
x 11101
y 11110
z 11111
cost: 3304
